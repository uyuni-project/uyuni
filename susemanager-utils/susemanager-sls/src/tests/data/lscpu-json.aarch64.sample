Architecture:        aarch64
Byte Order:          Little Endian
CPU(s):              64
On-line CPU(s) list: 0-63
Thread(s) per core:  1
Core(s) per socket:  4
Socket(s):           16
NUMA node(s):        4
Vendor ID:           ARM
Model:               2
Model name:          Cortex-A72
Stepping:            r0p2
BogoMIPS:            100.00
NUMA node0 CPU(s):   0-15
NUMA node1 CPU(s):   16-31
NUMA node2 CPU(s):   32-47
NUMA node3 CPU(s):   48-63
Flags:               fp asimd evtstrm aes pmull sha1 sha2 crc32 cpuid
