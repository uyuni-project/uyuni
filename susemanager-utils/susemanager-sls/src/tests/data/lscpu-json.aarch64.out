{
    "cpu_model": "Cortex-A72",
    "cpu_vendor": "ARM",
    "cpu_numanodes": "4",
    "cpu_sockets": "16",
    "cpu_stepping": "r0p2",
    "cpu_sum": "64",
    "cpu_threads": "1",
    "cpu_cores": "4"
}
